src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1025,6 +1026,12 @@ struct SmartAction
             uint32 wp6;
         } closestWaypointFromList;
 
+        struct
+        {
+            uint32 sound[5];
```

Comment:
Hmm, why does this use 5, but here in the loop its not: https://github.com/Lopfest/TrinityCore/commit/f2e573bd34a894fd9249e124c7b6f37acb0a8aef#diff-656b21c7502ec846ec8202b538a79321R832
(I mean it evaluates to 5 though)?

Maybe they could be unified?